docs/source/getting_started.rst

```diff
@@ -1,2 +1,155 @@
-Getting Started in 5 Minutes
-============================
+##############################
+Getting Started with Orchestra
```

Comment:
I'm skeptical that this will be true, but added it in because it's in that spirit :ghost: